### PR TITLE
Moved examples to backend, forced /r in text files + fixed DebugInfo

### PR DESCRIPTION
**Pull request by nulls**

### CODE REVIEW

File: save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt [hunk 1]
```diff
@@ -217,7 +218,7 @@ class SandboxController(
         .flatMap { key ->
             storage.overwrite(
                 key = key,
-                content = Flux.just(ByteBuffer.wrap(content.toByteArray()))
+                content = Flux.just(ByteBuffer.wrap(content.replace("\r\n", "\n").toByteArray()))
```
Comment (sanyavertolet) [inline save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:221]: What about macOS and its `\r`?

**Review by sanyavertolet — APPROVED** [review]
lgtm, but pls think about `\r`

File: save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:221]: @Cheshiriks The point is that the file is sent from some PC to our sandbox, which is LF. We need to convert all possible eol symbols (CRLF, LF, CR) to LF

Comment (akuleshov7) [reply save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:221]: > System.lineSeparator() ?

backend is ran on ubuntu - System.lineSeparator - is always /n :)

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt [hunk 2]
```diff
@@ -347,8 +346,7 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
         if (response.ok) {
             response.text().await()
         } else {
-            postContentAsText(urlPart, fileName, defaultValue)
-            defaultValue
+            "No content for $fileName"
```
[the remark this answers is not in the document]
Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt:349]: return error message from response

File: save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt (hunk 1 above)
Comment (nulls) [reply save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt:221]: changed to `\r\n?` -- it should handle Win and Mac

**Pull request merged**